src/main/java/testapp/core/User.java

```diff
@@ -1,30 +1,40 @@
 package testapp.core;
 
-public class User {
-    private int id;
+import com.google.appengine.api.datastore.Entity;
+
+import java.io.Serializable;
+
+public class User implements Serializable {
+    private long ucn;
     private String name;
```

Comment:
I have to ask, why don't we have the id of the user in here, how are we going to extract it from the database in order to set it to the UserAccount, or in other words how do we recognize which account to which user belongs